info/erste_woche/Ersti-Kalender_tablesgenerator-com.tgn

```diff
@@ -0,0 +1 @@
cspan":1,"rspan":1,"markup":[]}],[{"value":"14 - 15","cspan":1,"rspan":1,"markup":[1,7]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"Ersti-","cspan":1,"rspan":1,"markup":[1,6]}],[{"value":"15 - 16","cspan":1,"rspan":1,"markup":[1,7]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"Kastenlauf","cspan":1,"rspan":1,"markup":[1,10]}],[{"value":"16 - 17","cspan":1,"rspan":1,"markup":[1,7]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]}],[{"value":"17 - 18","cspan":1,"rspan":1,"markup":[1,7]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]}],[{"value":"18 - 19","cspan":1,"rspan":1,"markup":[1,7]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]}],[{"value":"19 - 20","cspan":1,"rspan":1,"markup":[1,7]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]}],[{"value":"20 - 21","cspan":1,"rspan":1,"markup":[1,7]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]}],[{"value":"21 - 22","cspan":1,"rspan":1,"markup":[1,7]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]},{"value":"","cspan":1,"rspan":1,"markup":[]}]]},"theme":null,"fixed_layout":false,"markup":{"instances":[{},{"style":{"fontWeight":"","fontStyle":"","textDecoration":"","color":"","backgroundColor":""}},null,{"style":{"fontWeight":"400","fontStyle":"normal","textDecoration":"","color":"","backgroundColor":""}}]},"options":{"table_caption":"","table_label":"tab:my-table"}}
```

Comment:
Does this file need to be included in the Git repo?

Reply:
Yes, because doing tables on tablesgenerator.com from scratch may not be as painful as doing them completely in TeX but still it is advisable to have a starting point